test/simulator/timestamp/src/connection_simulator.cpp

```diff
@@ -135,6 +141,23 @@ connection_simulator::query_timestamp(
     uint64_t ts;
     if (query_timestamp == "all_durable") {
         ts = _durable_ts;
+
+        for (auto &session : _session_list) {
+            if (session->is_txn_running()) {
```

Comment:
For readability, it's preferred to do:
```suggestion
            if (!session->is_txn_running()) continue;
```

Reply:
s_all won't allow, reverting these changes, see [88c2d1c](https://github.com/wiredtiger/wiredtiger/pull/8398/commits/88c2d1c196ccd2e39dce10e4fdbc8d552469d4c6)

Reply:
I am pretty sure it would? `s_all` should block us from implementing this kind of early exit. What is the issue?

Reply:
Updated in [30871ab](https://github.com/wiredtiger/wiredtiger/pull/8398/commits/30871abc792bc0de99005cb8a734ccda1db3f27e)